OpenRA.Mods.D2k/Traits/MakesNoise.cs

```diff
@@ -0,0 +1,36 @@
+#region Copyright & License Information
+/*
+* Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+* This file is part of OpenRA, which is free software. It is made
+* available to you under the terms of the GNU General Public License
+* as published by the Free Software Foundation. For more information,
+* see COPYING.
+*/
+#endregion
+
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.D2k.Traits
+{
+	[Desc("This actor makes noise, which causes them to be targeted by actors with the Sandworm trait.")]
+	public class MakesNoiseInfo : ITraitInfo
```

Comment:
I think I mentioned this earlier, but the argument I saw on the IRC logs over this description reinforces it: if you need a description to explain the basic functionality of a trait, then you probably need to rethink the name.  `AttractsWorms` would be a good one for this.